plugins/@grouparoo/salesforce/src/lib/export/mapping.ts

```diff
@@ -194,7 +199,10 @@ const extractFields = (
 
     if (!field.nillable && !field.defaultedOnCreate) {
       // needs to be set to create
-      required.push({ key, type });
+      const syncMode = await destination.getSyncMode();
+      if (syncMode !== DestinationSyncModeData.enrich.key) {
+        required.push({ key, type });
+      }
```

Comment:
This seems to be the main logical change

Reply:
I wonder if we should pass in the this stuff from core to be more like `exportRecords` instead of the implicit stuff hanging off the Destination. Is that a good idea @evantahler ?
```
export interface DestinationSyncOperations {
    create: boolean;
    update: boolean;
    delete: boolean;
}
```